microservices/site-launch/lambda-functions/step-functions-trigger/index.ts

```diff
@@ -18,17 +20,17 @@ export const stepFunctionsTrigger = async (event: MessageBody) => {
       input: JSON.stringify(event),
     }
 
-    stepFunctions.startExecution(params, (res, err) => {
-      console.log(`Your state machine ${stateMachineArn} executed successfully`)
+    stepFunctions.startExecution(params, (err, res) => {
       if (err) {
         throw err
       }
+      console.log(`Your state machine ${stateMachineArn} executed successfully`)
```

Comment:
if we wanna keep this, we probably wanna change it to `logger.info`!

Reply:
3773fbf